static/app/utils/fields/index.ts

```diff
@@ -341,6 +341,14 @@ function validateForNumericAggregate(
   };
 }
 
+function getDynamicFieldValueType() {
```

Comment:
Does this need to be wrapped in a function?

Reply:
Woops @malwilley, I fixed it